.githooks/pre-commit

```diff
@@ -7,28 +7,28 @@ function check_docker_vars
 {
     grep -n '\-[[:space:]]'"${1}"'=[^$]' docker/deploy/docker-compose.yml
     if [[ "$?" == 0 ]]; then 
-        echo "Please remove the custom settings for ${1} from docker/deploy/docker-compose.yml"
-        echo "The line should be:"
-        echo "#- ${1}="
+        printf "\e[31mPlease remove the custom settings for ${1} from docker/deploy/docker-compose.yml\n"
+        printf "The line should be:\n"
+        printf "#- ${1}=\e[0m\n"
```

Comment:
You shouldn't change docker-compose at all. We have a Powershell script that sets the proper environment variables and runs docker-compose. It's not in the repo since it contains credentials to one of our company mailboxes.